internal/resourcepolicies/volume_resources.go

```diff
@@ -133,6 +133,10 @@ func (c *csiCondition) match(v *structuredVolume) bool {
 		return true
 	}
 
+	if c.csi.Driver == "" { // match csi: {}
+		return v.csi != nil
+	}
+
 	if v.csi == nil {
 		return false
 	}
```

Comment:
Could you add unit tests for the new code?

Reply:
Added

Reply:
Added